tools/build_api.py

```diff
@@ -176,7 +176,7 @@ def build_project(src_path, build_path, target, toolchain_name,
 
     if report != None:
         start = time()
-        
+
```

Comment:
This change mixes white space changes (i.e. formatting) with functional changes.

It should be split into two different commits.  One with the none functional changes (the formatting) the other with the functional change.